### DISCUSSION TITLE
May we have a plot file check (scan) tool?

### DISCUSSION THREAD

**Discussion by overdoignism** [Chia-Network/chia-blockchain#7690] [Ideas]
Because some reason, eg. Bad RAM/SSD/Lan ....

The plot file may be corrupted.

Only we can do is waiting chia tool found it when use.

May we have a tool for plot file check and scan? 

Thanks.

**Comment by d1Mm** [comment]
you can use CLI command [**chia plots check**](https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference#check)
